### PR TITLE
Deploy in wechat4u

### PR DESCRIPTION
**Pull request by abhishek-iiit**
## Added deploy in wechat4u
Screenshot from 2021-07-29 20-17-29
Screenshot from 2021-07-29 20-17-38
Screenshot from 2021-07-29 20-17-50

### CODE REVIEW

**Comment by abhishek-iiit** [comment]
Linked to issue #896

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by vasvi-sood — APPROVED** [review]
LGTM!

File: docusaurus/docs/howto/deploy-wechat4u.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+title: Deploy in WeChat4U
+---
+
+WeChat4U is an excellent wechat bot framework that supports both Node.js & Browser, with rich features and an active community of experienced contributors. In this how to guide, you will learn how to deploy the bot in WeChat4U locally and integrate bot from the list of examples present in [starter templete](https://github.com/wechaty/wechaty-getting-started).
```
Comment (proudofsimin) [inline docusaurus/docs/howto/deploy-wechat4u.mdx]:
```suggestion
WeChat4U is an excellent wechat bot framework that supports both Node.js and Browser, with rich features and an active community of experienced contributors. In this how to guide, you will learn how to deploy the bot in WeChat4U locally and integrate bot from the list of examples present in [starter templete](https://github.com/wechaty/wechaty-getting-started).
```

File: docusaurus/docs/howto/deploy-wechat4u.mdx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+![Deploy in WeChat4U](../../static/img/docs/howto/IM_platform/wechat4u.webp)
+
+## Requirements
+
+1. Your system must have [Node.js](https://nodejs.org/en/download/package-manager/) installed (version >= 12).
+2. Your system must have [Wechaty](https://github.com/wechaty/wechaty) (version >= 0.40).
+3. You must be familiar with [Wechaty puppet wechat4u](https://www.npmjs.com/package/wechaty-puppet-wechat4u).
+
+## Deployment
+
```
Comment (proudofsimin) [inline docusaurus/docs/howto/deploy-wechat4u.mdx:16]: Run `wechaty-puppet-wechat4u`:

**Review by proudofsimin — APPROVED** [review]
Use "and" instead of "&" could make your document more professional.

File: docusaurus/docs/howto/deploy-wechat4u.mdx (hunk 2 above)
Comment (abhishek-iiit) [reply docusaurus/docs/howto/deploy-wechat4u.mdx:16]: Thankyou Simin, added the changes

**Review by huan — APPROVED** [review]
LGTM.

```diff
- docusaurus/static/img/docs/howto/IM_platform/wechat4u.webp
+ docusaurus/static/img/docs/howto/deploy-wechat4u/wechat4u.webp
```

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Comment by Rohitesh-Kumar-Jain** [comment]
@vasvi-sood and @abhishek-iiit pls solve the merging conflict, and help yourself in merging the PR

**Comment by abhishek-iiit** [comment]
Merging PR with no conflicts and 4 approvals :)

**Pull request merged**